allure-pytest/src/plugin.py

```diff
@@ -102,6 +102,20 @@ def a_label_type(string):
                                          help="""Comma-separated list of IDs.
                                          Run tests that have at least one of the specified id labels.""")
 
+    def cf_type(string):
+        type_name, *values = string.split("=")
+        atoms = set(values[0].split(","))
```

Comment:
Better to split by the first `=` only - less restrictions on label values (label values may contain `=` in that case).
```suggestion
        type_name, values = string.split("=", 1)
        atoms = set(values.split(","))
```